src/controllers/update/update.beneficial.owner.gov.controller.ts

```diff
@@ -17,7 +22,3 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
 export const update = (req: Request, res: Response, next: NextFunction) => {
   return updateBeneficialOwnerGov(req, res, next, UPDATE_BENEFICIAL_OWNER_TYPE_URL, false);
 };
-
-export const remove = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This remove points (same for others) have definitely been confirmed not to be required? BA?

Reply:
Yeah don't need BA confirmation as I don't think they are aware it was even brought over from the registration journey in the first place haha! We copied and pasted everything from registration when initially creating the Update BO controllers assuming that they would all be used but the 'remove' endpoints here aren't in use.
And this new confirm.to.remove controller will handle the removing

Reply:
All the removes go via a new "do you want to remove" link so I think so